contracts/core/FollowNFT.sol

```diff
@@ -105,39 +108,41 @@ contract FollowNFT is LensNFTBase, IFollowNFT {
         override
         returns (uint256)
     {
-        if (blockNumber > block.number) revert Errors.BlockNumberInvalid();
+        unchecked {
```

Comment:
The `getPowerByBlockNumber` and `getDelegatedSupplyByBlockNumber` can be simplified to use one underlying helper function. Both are essentially looking in a `mapping(uint256 => Snapshot)` so passing `_snapshots[user]` or `_delSupplySnapshots` should let us use just one underlying.

Reply:
Resolved and merged to main in your PR #84!